tests/e2e/pageobjects/ide/Editor.ts

```diff
@@ -338,36 +337,18 @@ export class Editor {
     async waitStoppedDebugBreakpoint(tabTitle: string, lineNumber: number, timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
         Logger.debug(`Editor.waitStoppedDebugBreakpoint title: "${tabTitle}" line: "${lineNumber}"`);
 
-        const stoppedDebugBreakpointLocator: By = By.xpath(await this.getStoppedDebugBreakpointXpathLocator(tabTitle, lineNumber));
-        await this.driverHelper.waitVisibility(stoppedDebugBreakpointLocator, timeout);
+        await this.driverHelper.waitUntilTrue(() => this.isBreakpointPresent(lineNumber, true), timeout);
```

Comment:
and I guess missed "await"
```
await this.driverHelper.waitUntilTrue(await () => this.isBreakpointPresent(lineNumber, true), timeout);
```

Reply:
Putting await on that place is not good (compilation error).
Only thing how we could rewrite it would be like this:
```
await this.driverHelper.waitUntilTrue(async () => await this.isBreakpointPresent(lineNumber, true), timeout);
```
But that doesn't make much sense to me, because `isBreakpointPresent` returns `Promise<boolean>`. Await waits for the promise resolution -> making it `boolean` and then, the async will make this inline function to return `Promise<boolean>` once again...
From this it seems to me, that writing it without async-await is exactly the same as writing it with async-await.